images/primary_node/Dockerfile

```diff
@@ -47,3 +47,20 @@ RUN yum -y install java-1.8.0-openjdk-devel \
     mapr-webserver \
     mapr-zookeeper \
     mysql-server
+
+RUN if [ "${MAPR_VERSION}" == "6.0.0-rc1" ] ; then \
```

Comment:
`==` should actually be `=` ([Bash is weird](http://tldp.org/LDP/abs/html/comparison-ops.html))